evalai/challenges.py

```diff
@@ -54,3 +71,15 @@ def future():
     Invoked by running `evalai challenges future`
     """
     display_future_challenge_list()
+
+
+@challenge.command()
+@click.pass_obj
+@click.argument('TEAM', type=int)
+def participate(ctx, team):
+    """
+    Participate in a challenge.
+
+    Invoked by running `evalai challenge CHALLENGE participate TEAM`
+    """
```

Comment:
**One issue:** 
In the current docstring, the `--help` command isn't able to display the complete docstring.
**Fix:**
Increase the word limit of `--help` command. Please check this if we can alter this. if not, then change the docstring to this format.
```
"""
Participate in a challenge
"""
"""
Invoked by running `evalai challenge CHALLENGE participate TEAM`
"""
```
The first comment here will be used by the `--help` command whereas second one will help any other developer or user.

Reply:
Actually, I used the current style because it did the exact functionality that you mentioned. Should I change it?

Reply:
Please check once. 
For more reference please see the screenshot below
<img width="736" alt="screen shot 2018-06-24 at 1 07 28 am" src="https://user-images.githubusercontent.com/12206047/41813026-1e543c86-774b-11e8-8d2c-7d4a95bab4d9.png">